### PR TITLE
pybind/mgr/restful: fix typo

### PR DESCRIPTION
**Pull request by nrdmn**
Fixed typo

### CODE REVIEW

**Comment by hjwsm1989** [comment]
please have look at this https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst

File: src/pybind/mgr/restful/api/__init__.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def get(self, **kwargs):
         return {
             'api_version': 1,
             'auth':
-                'Use "ceph restful create_key <key>" to create a key pair, '
+                'Use "ceph tell mgr restful create-key <key>" to create a key pair, '
```
Comment (liewegas) [inline src/pybind/mgr/restful/api/__init__.py]: no need for 'tell mgr' now

**Comment by liewegas** [comment]
Can you squash this down to one commit that just fixes the type (_ to -)?  Thanks!

**Comment by nrdmn** [comment]
done! :)

**Comment by liewegas** [comment]
Thanks!

**Pull request merged**